### PR TITLE
fix(alerts): void query with numeric comparison

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
Fix for Alerts & Reports:

If an alert query returns no rows and the validation is a comparison, this fix assumes that no rows is zero and applies the configured comparison to that  

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: tests/reports/commands_tests.py [hunk 1]
```diff
@@ -342,6 +342,11 @@ def create_no_alert_email_chart(request):
             "validator_type": ReportScheduleValidatorType.NOT_NULL,
             "validator_config_json": "{}",
         },
+        "alert7": {
+            "sql": "SELECT first from test_table where first=0",
```
Comment (villebro) [inline tests/reports/commands_tests.py:346]: If this is supposed to return zero rows, would a more obvious false where clause be more descriptive? Something like `where 1 = 0`? The other examples above are easy to reason about as they have a static query that doesn't reference a table.

**Review by villebro — COMMENTED** [review]
An outsider's comment to help make the test more obvious

File: tests/reports/commands_tests.py (hunk 1 above)
Comment (dpgaspar) [reply tests/reports/commands_tests.py]: good point, better readability. Updated

**Review by villebro — APPROVED** [review]
LGTM 👍

**Review by willbarrett — APPROVED** [review]
I think ideally we'd throw a specific error when no data is returned, rather than defaulting to zero, but I don't want to block this change as it addresses a current pain-point.

**Pull request merged**